### PR TITLE
Retain Library/Boards Manager installation interface using version menu

### LINKED ISSUE #1425: Retain Library/Boards Manager installation interface following version menu selection
### Describe the request

Find some way to retain the installation interface of the target item when using the version menu in **Library Manager** and **Boards Manager**.

🙂 Library and Boards Manager will be easier and more pleasant to use.

### Describe the current behavior

The IDE's **Library Manager** and **Boards Manager** list the available libraries and boards platforms.

When the mouse pointer is hovered over one of the list items, an "**INSTALL**" button as well as a version menu for items that have multiple releases are added at the bottom of the item.

The version menu is a dropdown, which overlaps the next item on the list. This means that the mouse pointer is hovering over the coordinates of the next item when selecting a different version from the menu.

🙁 After selecting a version from the menu, the installation interface of the target item is lost and the installation interface for the next item on the list opens instead. Even though I am very experienced using the IDE, I still find myself accidentally installing the wrong library or simply feeling confused by the unexpected change in the interface when I use this menu. This makes me suspect that it will be a poor user experience for those who are less experienced.

version-menu

### Arduino IDE version

2.0.0-rc9.4

### Operating system

Windows, Linux

### Operating system version

Windows 10, Ubuntu 20.04

### Issue checklist

- [X] I searched for previous requests in [the issue tracker](https://github.com/arduino/arduino-ide/issues?q=)
- [X] I verified the feature was still missing when using the latest [nightly build](https://github.com/arduino/arduino-ide#nightly-builds)
- [X] My request contains all necessary details

### PR DESCRIPTION
**Pull request by francescospissu**
### Motivation
If the version selected from the drop-down menu overlaps the next element, the focus on the current element is lost and the "INSTALL" button is no longer visible. Details [here](https://github.com/arduino/arduino-ide/issues/1425).

### Change description
Keep the footer visible until installation is performed. (NOTE: The selection from the drop-down menu is not visible in the video because it wasn't captured by the screen recorder).

https://user-images.githubusercontent.com/94986937/194260064-b45b8e55-3897-4c11-b101-1b825fe12e70.mp4

### Other information
Closes #1425.

### Reviewer checklist

* [ ] PR addresses a single concern.
* [ ] The PR has no duplicates (please search among the [Pull Requests](https://github.com/arduino/arduino-ide/pulls) before creating one)
* [ ] PR title and description are properly filled.
* [ ] Docs have been added / updated (for bug fixes / features)

### CODE REVIEW

**Review by per1234 — APPROVED** [review]
This is super nice. Thanks Francesco!

**Review by AlbyIanna — APPROVED** [review]
Looks good to me, both code and behaviour, but let's wait for an approval from @91volt before merging this PR.

**Review by 91volt — APPROVED** [review]
lgtm 👍

**Pull request merged**
